addons/folo/common/src/main/java/org/commonjava/indy/folo/ctl/FoloAdminController.java

```diff
@@ -454,6 +454,11 @@ private TrackedContentEntry recalculate( final TrackedContentEntry entry )
                 TransferOperation.UPLOAD :
                 TransferOperation.DOWNLOAD );
 
+        if ( transfer == null )
```

Comment:
Will this result in the tracking record losing information if those entries get promoted to a consolidated hosted repo or something? It seems like it might be safer to return the partial information that was passed in, instead of null...?

Reply:
Yeah, agreed.